src/utils/web3_tools.js

```diff
@@ -110,3 +110,14 @@ export async function deployContract(web3, abi, bytecode, args = [], options = {
       ...options
     });
 }
+
+export async function deploy(web3, contractJson, args = []) {
+  const contract = await deployContract(web3, contractJson.abi, contractJson.bytecode, args);
+  return contract.options.address;
```

Comment:
I don't see a lot of value in this function + the name is confusing as the difference between `deploy` and `deployContract` it is not obvious.

Reply:
done 👍